lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -869,6 +869,81 @@ namespace Js
     }
 
     template <typename T>
+    BOOL DictionaryTypeHandlerBase<T>::DeleteProperty(DynamicObject *instance, JavascriptString *propertyNameString, PropertyOperationFlags propertyOperationFlags)
```

Comment:
Well, it is really the counterpart of DeleteProperty which takes PropertyId as parameter.

---
In reply to: [78791124](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r78791124) [](ancestors = 78791124)